source/developers-guide/shopware-5-plugin-license/index.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: default
+title: Developer's Guide
+github_link: developers-guide/shopware-5-plugin-license/index.md
+---
+
+## Overview
+
+Should you wish to, you can have your plugin run a license check when used in a Shopware shop. This will ensure that the target shop has permission to use your plugin. This document covers the steps necessary to implement this check. 
+
+## Requesting the license check
+
+To request a license check, you must first login into your [Shopware account](http://account.shopware.com) and request a license validation in the detail page of your plugin. Once you have done so, you will be given a code snippet similar to this: 
+
+```
```

Comment:
Remove the indentation. Code fences are enough. See comments in the original PR.